### PR TITLE
to compile with gcc5x

### PR DESCRIPTION
**Pull request by KjellKod**

### CODE REVIEW

File: LINUX/scripts/buildRpm.sh [hunk 1]
```diff
@@ -23,6 +23,14 @@ rpmdev-setuptree
 KERNEL=`uname -r`
 sed -e 's/@KERNEL_VERSION@/'${KERNEL%.x86_64}'/g'  packaging/netmap.spec > ~/rpmbuild/SPECS/netmap.spec
 
+# Necessary step to allow gcc5 to build netmap
+# ref: http://stackoverflow.com/questions/29925513/compile-a-linux-2-6-kernel-module-with-newer-compiler
+
+GCC5_KERNEL_FILE="/usr/src/kernels/${KERNEL}/include/linux/compiler-gcc5.h"
+echo "kernel file: $GCC5_KERNEL_FILE"
+sudo touch $GCC5_KERNEL_FILE
+echo "#include <linux/compiler-gcc4.h>" | sudo tee --append "$GCC5_KERNEL_FILE"
```
Comment (john-gress) [inline LINUX/scripts/buildRpm.sh]: You need to check if the file exist. If it does don't add to it.

Comment (KjellKod) [inline LINUX/scripts/buildRpm.sh]: good point

**Comment by john-gress** [comment]
:+1:

**Comment by KjellKod** [comment]
verified with Craig

**Pull request merged**